src/utils/apis.js

```diff
@@ -22,6 +21,17 @@ export function getArticles(params) {
     .catch((err) => console.log(err, 'all articles'));
 }
 
+export function getSingleArticle({article_id}) {
+  return axios
+  .get(`${baseURL}articles/${article_id}`)
+  .then(({ data }) => {
+    return data;
+  })
+  .catch((err) => console.log(err, 'individual article'));
```

Comment:
Nice way to make sure you know where the error is coming from by putting the string in the log of the error